unity/Assets/0Refactored/scenes/experiments/HuygensPrinciple/scripts/InterferencePlate.cs

```diff
@@ -18,15 +19,27 @@ private void Start()
 
         private void Update()
         {
-            //ToDo: Add OnMovement Event to the SlitPlate script
+        }
+
+        protected override void HandleUpdate()
+        {
+        }
+
+        protected override void HandleFixedUpdate()
+        {
             if (slitPlate.transform.hasChanged)
```

Comment:
I think we can combine these two conditions.
e.g.: if(!slitPlate.transform.hasChanged && !_parameterHasChanged) return;